fvm/evm/stdlib/contract.cdc

```diff
@@ -442,10 +442,11 @@ contract EVM {
         }
 
         /// Withdraws the balance from the cadence owned account's balance
-        /// Note that amounts smaller than 10nF (10e-8) can't be withdrawn
+        /// Note that amounts smaller than 1e10 attoFlow can't be withdrawn
         /// given that Flow Token Vaults use UFix64s to store balances.
         /// If the given balance conversion to UFix64 results in
-        /// rounding error, this function would fail.
+        /// rounding loss, the withdrawal amount will be truncated to the
+        /// maximum precision for UFix64.
```

Comment:
We need to mention the `ErrWithdrawBalanceRounding` case, otherwise, it would look like the transaction will succeed without withdraw any amount.

Reply:
Good point :+1:  Updated in https://github.com/onflow/flow-go/pull/6877/commits/fb356030010e44ba020f716cebc4b89c7e497dc4 .